### DISCUSSION TITLE
✨ [Feature]: 封装通用的响应式组件

### DISCUSSION THREAD

**Discussion by kagol** [DevCloudFE/vue-devui#1374] [General]
### What problem does this feature solve

封装通用的响应式组件，这样其他组件可以直接使用，在不同分辨率下都能友好地显示

### What does the proposed API look like

待讨论

**Comment by GaoNeng-wWw** [comment]
可以封装成指令，方便其他组件复用

**Comment by kagol** [comment]
> 可以封装成指令，方便其他组件复用

可以的，要想下怎么设计比较合适

**Comment by GaoNeng-wWw** [comment]
> > 可以封装成指令，方便其他组件复用
> 
> 可以的，要想下怎么设计比较合适

可以封装成```@v-media```，读取根目录的某一个文件(比如:```vue-devui.config.ts/vue-devui.config.js```)。如果文件为空则使用默认断点
